contracts/src/v0.8/tests/VRFConsumer.sol

```diff
@@ -23,7 +23,7 @@ contract VRFConsumer is VRFConsumerBase {
     requestId = requestId;
   }
 
-  function testRequestRandomness(bytes32 keyHash, uint256 fee) external returns (bytes32) {
+  function doRequestRandomness(bytes32 keyHash, uint256 fee) external returns (bytes32) {
```

Comment:
nit: you call it just 'requestRandomness' in the V2 Consumer

Reply:
This is the V1 consumer actually. `requestRandomness` is already defined in `VRFConsumerBase` and is marked `internal`, so you need this kind of `external` wrapper function unfortunately.